test/ejabberd_tests/tests/cluster_commands_SUITE.erl

```diff
@@ -134,3 +152,53 @@ set_master_test(ConfigIn) ->
     ejabberdctl("set_master", ["self"], ConfigIn),
     [MasterNode] = rpc_call(mnesia, table_info, [TableName, master_nodes]).
 
+join_successful(Config) ->
+    Node2 = ct:get_config(ejabberd2_node),
```

Comment:
I'm going also to stop using config values like `ejabberd2_node` and `ejabberd_node` and convert suite to use `hosts` block defined here [1]

[1]https://github.com/esl/MongooseIM/blob/master/test/ejabberd_tests/test.config#L26